src/main/scala-2.11/org/apache/james/gatling/control/UserFeeder.scala

```diff
@@ -1,30 +1,59 @@
 package org.apache.james.gatling.control
 
-import java.util.UUID
-
 import io.gatling.core.Predef._
 
+import scala.concurrent.duration.Duration.Inf
+import scala.concurrent.{Await, Future}
 import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
+import scala.util.Success
 
 object UserFeeder {
 
-  def createUserFeeder(userCount: Int): Array[Map[String, String]] = {
-    val domain = Domain(UUID.randomUUID().toString)
+  type UserFeeder = Array[Map[String, String]]
+
+  val USERNAME_SESSION_PARAM: String = "username"
+  val PASSWORD_SESSION_PARAM: String = "password"
+
+  def createUserFeeder(userCount: Int): UserFeeder =
```

Comment:
why not a Future[UserFeeder] ? We could let the caller wait if it wants to, don't you think ?